ebpf-io/ebpf.io-website
# feat: adds french language

**Describe what changes this pull request brings**
This pull request adds French language.


**Useful links:**
[Link to the language selection design in the footer in Figma](https://www.figma.com/file/DKNPEcutVbDZb1ylyP7wHs/EBPF-Website-Design?type=design&node-id=3324%3A45867&mode=design&t=CYnIqgPO1l7g19LB-1)


**Steps to test:**
1. Open the [index page](https://deploy-preview-476--ebpfio.netlify.app/fr-fr/), [what-is-ebpf page](https://deploy-preview-476--ebpfio.netlify.app/fr-fr/what-is-ebpf/), [get-started page](https://deploy-preview-476--ebpfio.netlify.app/fr-fr/get-started/), [case-studies page](https://deploy-preview-476--ebpfio.netlify.app/fr-fr/case-studies/), [contribute page](https://deploy-preview-476--ebpfio.netlify.app/fr-fr/contribute/)
1. Confirm that everything looks and works as expected


**TODO:**
- [x]  Language switch select in header
- [x]  Translation of header navigation
- [x]  Translation of footer navigation
- [ ]  ~~Translation all text for case-studies page~~
- [x]  There is no translation for the title and subtitle on the "case-studies" page
- [x]  On the main page in the first block there is no translation for the "Project Landscape" and "What is eBPF" buttons
- [ ]  ~~Translation all text for what-is-ebpf page (On the "what is ebpf" page in the last block "Further Reading" only the headings are translated)~~
- [x]  Translation SEO meta tags
